web/blog/2022-09-05-dev-excuses-app-tutrial.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Building a backbone for a full-stack web app with a single config file.
```

Comment:
It seems the other titles do not end in punctuation. Maybe remove the period for consistency?